docs/getting-started/setup.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Setup
+---
+
+## Vanilla Setup
+
+Manually setting up pulse without a framework
+
+```js
+import Pulse from 'pulse-framework';
+
+new Pulse.Library({
```

Comment:
Vanilla Setup won't work without a basic storage implementation. Maybe we should add something like this

```
storage: {
    async: true,
    store: {},
    get: function(key) {
      return this.store[key];
    },
    set: function(key, value) {
      this.store[key] = value.toString();
    },
    clear: function() {
      this.store = {};
    },
    remove: function(key) {
      delete this.store[key];
    }
  },
```

Reply:
By default pulse uses the localStorage system for persisting data.
No storage object needs to be defined for it to work.
Example: https://codesandbox.io/s/pulse-vanilla-8x8og?expanddevtools=1&fontsize=14&module=%2Fsrc%2Findex.js

Reply:
You're right as long as you're web based. React native doesn't have the window.localStorage

Reply:
Very true. Though this example is for vanilla not React or ReactNative.
We'd greatly appreciate additions to the docs if you'd like to start the ReactNative section.

Reply:
Sounds good. I will write something together. Additions coming soon 👍

Reply:
Awesome; Thank you so much for the help! :smiley: